org.eclipse.tm4e.languageconfiguration/src/main/java/org/eclipse/tm4e/languageconfiguration/internal/utils/TextUtils.java

```diff
@@ -182,4 +182,28 @@ public static TabSpacesInfo getTabSpaces(@Nullable final ITextViewer viewer) {
 		return new TabSpacesInfo(4, false);
 	}
 
-}
+	/**
+	 * Determines if all the characters at any offset of he specified document range are space or tab characters.
```

Comment:
minor typo "of he specified" -> "of the specified"

Reply:
Thanks. I'll fix it.